src/Utility/Text.php

```diff
@@ -852,4 +859,60 @@ public static function parseFileSize($size, $default = false)
         }
         throw new InvalidArgumentException('No unit type.');
     }
+
+    /**
+     * Transliterate string.
+     *
+     * @param string $string String to transliterate.
+     * @param string|null $transliteratorId Transliterator identifer. If null
+     *   Text::$defaultTransliteratorId will be used.
+     * @return string
+     * @see http://php.net/manual/en/transliterator.transliterate.php
+     */
+    public static function transliterate($string, $transliteratorId = null)
+    {
+        $transliteratorId = $transliteratorId ?: static::$defaultTransliteratorId;
+        return transliterator_transliterate($transliteratorId, $string);
```

Comment:
@ravage84 Cause it was more convenient `transliterator_transliterate()` :stuck_out_tongue:  I can be changed to using a `Transliterator` instance and caching it in a static property to avoid recreating across calls.